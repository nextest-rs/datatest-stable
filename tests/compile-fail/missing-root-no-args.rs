datatest_stable::harness! {
    {
        test = my_test,
    },
}
