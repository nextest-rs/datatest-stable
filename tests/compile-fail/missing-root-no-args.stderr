error: expected `root`, but ran out of arguments (collected: test = my_test,)
 --> tests/compile-fail/missing-root-no-args.rs:1:1
  |
1 | / datatest_stable::harness! {
2 | |     {
3 | |         test = my_test,
4 | |     },
5 | | }
  | |_^
  |
  = note: this error originates in the macro `$crate::harness_collect` which comes from the expansion of the macro `datatest_stable::harness` (in Nightly builds, run with -Z macro-backtrace for more info)
